Source/DataSources/GroundGeometryUpdater.js

```diff
@@ -69,6 +98,19 @@ define([
         }
 
         this._zIndex = defaultValue(geometry.zIndex, defaultZIndex);
+
+        var heightProperty = geometry.height;
+        var extrudedHeightProperty = geometry.extrudedHeight;
+
+        if (this._terrainOffsetProperty instanceof TerrainOffsetProperty) {
```

Comment:
Shouldn't this just be a defined check?  When wouldn't `_terrainOffsetProperty` be anything other than an instance of `TerrainOffsetProperty`?